src/Choice Tile/index.js

```diff
@@ -0,0 +1,65 @@
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import { SIZES } from '../constants';
+import { wcBool } from '../utils';
+
+const ChoiceTile = ({
+  checked,
+  children,
+  className,
+  disabled,
+  icon,
+  id,
+  invalid,
+  onChange,
+  size,
+  subdued,
+  title,
+  ...rest
+}) => {
+  return (
+    <label className="hxChoice">
+      <input
+        checked={wcBool(checked)}
+        disabled={wcBool(disabled)}
+        invalid={wcBool(invalid)?.toString()}
```

Comment:
```suggestion
        checked={checked}
        disabled={disabled}
        invalid={invalid?.toString()}
```
might not need `wcBool` on elements that are not web components.

Reply:
While testing this out, the checked behavior only worked when using `wcBool`

Reply:
Ok. Thanks for clarifying.